src/Components/List/List.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import '../List/List.css';
+
```

Comment:
The css import can just be ./List since the css and jsx are in the same folder

Reply:
Fixed this, thank you!